wurst/systems/entities/bosses/AlligatorMan.wurst

```diff
@@ -2,7 +2,30 @@ package AlligatorMan
 import ID
 import Hostile
 import LinkedList
+import BossUtilities
+import RegisterEvents
+import HashMap
 
+//Regions for the alligator islands
+public let NW_ALLIGATOR_REGION = CreateRegion()
+    ..addRect(gg_rct_NW_Alligator)
+public let NE_ALLIGATOR_REGION = CreateRegion()
+    ..addRect(gg_rct_NE_Alligator)
+public let SW_ALLIGATOR_REGION = CreateRegion()
+    ..addRect(gg_rct_SW_Alligator)
+public let SE_ALLIGATOR_REGION = CreateRegion()
+    ..addRect(gg_rct_SE_Alligator)
+
+//List of alligator regions
+let REGIONS = asList(
+    NW_ALLIGATOR_REGION,
+    NE_ALLIGATOR_REGION,
+    SW_ALLIGATOR_REGION,
+    SE_ALLIGATOR_REGION
+)
+
+//Mapping of alligator regions to it's corresponding alligator
+var regionsToAlligators = new IterableMap<region, unit>()
```

Comment:
Does this not work as `let`? You need `var` to reassign the variable, but I think that modification via methods works fine with `let` because the reference itself is constant.